src/resources/UsageAnalytics/Read/Filters/Filters.ts

```diff
@@ -56,4 +66,97 @@ export default class Filters extends Resource {
     deleteReportFilter(id: string) {
         return this.api.delete(this.buildPath(`${Filters.reportingBaseUrl}/${id}`, {org: this.api.organizationId}));
     }
+
+    /**
+     * Get all permission filters
+     */
+    listPermissionFilters() {
+        return this.api.get<ListFiltersResponse>(
+            this.buildPath(Filters.permissionsBaseUrl, {org: this.api.organizationId})
+        );
+    }
+
+    /**
+     * Get a permission filter
+     */
+    getPermissionFilter(id: string) {
+        return this.api.get<FilterModel>(
+            this.buildPath(`${Filters.permissionsBaseUrl}/${id}`, {org: this.api.organizationId})
+        );
+    }
+
+    /**
+     * Create a permission filter
```

Comment:
Are these comments really helpful? Because the name of the methods are clear enough to me.

Reply:
Everything is pulled directly from the Swagger docs, we don't necessarily need them but I figured I would include all the descriptions from there